web/styles/settings.css

```diff
@@ -697,6 +697,14 @@ input[type="checkbox"] {
     }
 }
 
+.settings-profile-user-field.editable-date-field {
+    display: grid;
+    grid-template-areas: "datepicker close-button";
+    grid-template-columns: minmax(0, 1fr) 1.4285em; /* 20px at 14px em */
+    align-items: center;
+    width: 325px;
```

Comment:
I'm working on another PR to standardize widths for settings input/dropdown with a variable, so I'd update this there too.